File: AdminPanal/AdminPanal.Web/Modules/ServerTypes/AylaStoreModule/ReviewForm.ts
import { StringEditor, DateEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface ReviewForm {
    Name: StringEditor;
    Email: StringEditor;
    Description: StringEditor;
    EntryDate: DateEditor;
}

export class ReviewForm extends PrefixedContext {
    static formKey = 'AylaStoreModule.Review';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!ReviewForm.init)  {
            ReviewForm.init = true;

            var w0 = StringEditor;
            var w1 = DateEditor;

            initFormType(ReviewForm, [
                'Name', w0,
                'Email', w0,
                'Description', w0,
                'EntryDate', w1
            ]);
        }
    }
}

File: AdminPanal/AdminPanal.Web/Modules/AylaStoreModule/Review/ReviewForm.cs
using Serenity.ComponentModel;
using System;

namespace AdminPanal.AylaStoreModule.Forms;

[FormScript("AylaStoreModule.Review")]
[BasedOnRow(typeof(ReviewRow), CheckNames = true)]
public class ReviewForm
{
    public string Name { get; set; }
    public string Email { get; set; }
    public string Description { get; set; }
    public DateTime EntryDate { get; set; }
}